package.json

```diff
@@ -7,9 +7,10 @@
     "dist/backdraft.js"
   ],
   "scripts": {
+    "eslint": "eslint-git-changes --jsx",
     "setup": "npm install -g grunt-cli",
-    "specs": "karma start --single-run",
-    "dev": "karma start",
+    "specs": "yarn run eslint; karma start --single-run",
+    "dev": "yarn run eslint && karma start",
```

Comment:
How come we use `&&` here but `;` in the `specs` command?

Reply:
This should be a `;` as well...I will update it